azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1267,6 +1278,10 @@ func flattenKubernetesClusterAgentPoolProfiles(profiles *[]containerservice.Mana
 			"fqdn": fqdnVal,
 		}
 
+		if profile.OrchestratorVersion != nil {
+			agentPoolProfile["orchestrator_version"] = *profile.OrchestratorVersion
+		}
```

Comment:
in changes merged since this PR was opened we're now ensuring the fields in the nested object are set even if this is nil - as such could we make this:

```suggestion
		orchestratorVersion := ""
		if profile.OrchestratorVersion != nil {
 			orchestratorVersion = *profile.OrchestratorVersion
 		}
 		agentPoolProfile["orchestrator_version"] = orchestratorVersion
```

Reply:
Fine with that :)